### ISSUE TITLE
Screen-Sreach, navigate on screen without mouse

### ISSUE THREAD

**Issue by DRYN07** [Flow-Launcher/Flow.Launcher#2826]
Fluent-Search App has an awesome feature called "Search in-window" where if you press a shortcut, you can navigate on the screen with keyboard. I think this would be an amazing feature to have in Flow Launcher.

![image](https://github.com/Flow-Launcher/Flow.Launcher/assets/29770761/e07faef2-c310-4dc5-825a-04ed43127404)

**Comment by DRYN07** [comment]
@verajosemanuel yes, I do the same exactly. Fluent search has great potential but Flow Launcher is much more convenient to use. I also use this feature from fluent search.

**Comment by verajosemanuel** [comment]
@DRYN07 i have found an excellent tool (better than fluent screen hints) : https://pit-ray.github.io/win-vind/

**Comment by DRYN07** [comment]
> [@DRYN07](https://github.com/DRYN07) i have found an excellent tool (better than fluent screen hints) : https://pit-ray.github.io/win-vind/ The hints position are much more accurate just in "tiny" settings mode is enough

Wow, this seems amazing, thanks for sharing it :)
